acsys/status.py

```diff
@@ -357,38 +429,73 @@ def __init__(self):
 ACNET_PEND = Status.create(Status.Codes.ACNET_PEND)
 ACNET_ENDMULT = Status.create(Status.Codes.ACNET_ENDMULT)
 
-# TODO deprecate this globals
-ACNET_RETRY = Status.create(Status.Codes.ACNET_RETRY)
+# TODO these globals can be used but are redundant.
+# Should be deleted on subsequent versions
+ACNET_RETRY = Status.create(Status.Codes.ACNET_RETRY) 
+""" @deprecated: Use AcnetRetryIOError() instead """
```

Comment:
If `.create` is to no longer be used the you can use the `warnings` library, https://docs.python.org/3/library/warnings.html, to issue a `DeprecationWarning`, https://docs.python.org/3/library/exceptions.html#DeprecationWarning when `.close` is called.

Reply:
mmm... you still should be able to call `Status.create(val)` when you don't know the value, or `AcnetRetryIOError()` when you want to be explicit, right?

Reply:
If we still want `Status.create` available then we should remove the `@deprecated` messages. If it's deprecated then we should use the `DeprecatedWarning`. To be clear the `warnings` library shouldn't disrupt code, just provide a warning.

Reply:
Once again I misunderstood. This is fine. I did find a package that might be able to formalize this, but I don't feel passionate that we should use it. https://pypi.org/project/deprecation/